build/configs/imxrt1050-evk/imxrt1050-evk_download.sh

```diff
@@ -114,28 +128,79 @@ function get_executable_name()
 	esac
 }	
 
+##Utility function to get partition index ##
+function get_partition_index()
+{
+        case $1 in
+                kernel | Kernel | KERNEL) echo "0";;
+                app | App | APP) echo "1";;
```

Comment:
We will not use `app` region anymore.

Reply:
This is necessary with protected build.

Reply:
Ok. Then how about using user instead of app?

Reply:
As we talked on chat, I agree but it is different kind of change. Let's do that with another PR.